### PR TITLE
docs: changelog 10-11-2024

### PR DESCRIPTION
**Pull request by danieltprice**
Preview:
https://neon-next-git-dprice-10-11-2024-changelog-neondatabase.vercel.app/docs/changelog

### CODE REVIEW

File: content/changelog/2024-10-11.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+- **Postgres 17 wait events system view support**: Added support for the new [pg_wait_events](https://www.postgresql.org/docs/current/view-pg-wait-events.html) system view introduced in Postgres 17. This view reports wait event types, useful for adding descriptions to wait events reported in `pg_stat_activity`.
+
+- **Extension update**: Updated the `neon` extension to version 1.5 for all Postgres versions.
+
+- **Neon API change**: The [Delete a project](https://api-docs.neon.tech/reference/deleteproject) endpoint now returns a `404 Not Found` response instead of a `200 OK` response if the project has already been deleted. This is a potentially breaking change for applications that expect a `200 OK` response for all delete operations, regardless of whether a project was actually deleted.
+
+- **Fixes**:
+  - Fixed an issue that prevented deleting a branch with an ephemeral compute endpoint created for performing a schema diff.
+  - Fixed an issue with the GitHub integration that prevented information in the integration drawer from updating after performing an action in the drawer.
```
Comment (bgrenon) [inline content/changelog/2024-10-11.md:56]:
```suggestion
  - Fixed an issue where the GitHub integration drawer wouldn’t update after changes were made.
```

**Review by bgrenon — APPROVED** [review]
Looks good!

File: content/changelog/2024-10-11.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+### Drag-to-zoom monitoring graphs
+
+We've enhanced the graphs on our **Monitoring** page to support drag-to-zoom. You can now click and drag to zoom in on a selected range.
+
+![drag-to-zoom monitoring graphs](/docs/relnotes/drag-to-zoom.png)
+
+### Don't forget to grab your ticket for Neon Deploy
+
+Join us on **October 30th** at **10:00 AM** PT to learn how Neon helps developers ship faster with Postgres. [Get your ticket now!](https://neon.tech/deploy)
```
Comment (ruf-io) [inline content/changelog/2024-10-11.md:23]: This part not needed - we will add to email.

```suggestion
```

File: content/changelog/2024-10-11.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  - Removed the previously deprecated `--allow-list` and `--ip-primary-only` options from the [projects update](/docs/reference/cli-projects#update) command. Operations performed by these options are supported by the [ip-allow](/docs/reference/cli-ip-allow) command.
+  - Removed the previously deprecated `--primary-only` option from the [ip-allow](/docs/reference/cli-ip-allow) command. It was replaced by the [ip-allow add --protected-only](https://neon.tech/docs/reference/cli-ip-allow#add) option.
+  - Updated the [project list](/docs/reference/cli-projects#list) command to output a user-friendly message when there are no projects or shared projects to display.
+
+  To update your Neon CLI installation to the latest version, follow our [CLI upgrade instructions](/docs/reference/cli-install#upgrade).
+
+- **Create Support Ticket modal enhancements**: We've made a few updates to the **Create support ticket** modal in the Neon Console:
+
+  - Added a drop-down menu for selecting a personal account or organization.
+  - Added support for selecting a shared project.
```
Comment (PlayLikeNeverB4) [inline content/changelog/2024-10-11.md:41]: I think support tickets for shared project was available already.

File: content/changelog/2024-10-11.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  - To help resolve support cases faster, the consent option to allow Neon Support staff to connect to your database is now selected by default. You can leave this option selected or deselect it when opening a support ticket.
+
+- **Organizations and Collaborators update**: When a collaborator is added as a member of an organization, they are now automatically removed as a collaborator from projects within that organization to avoid redundancy.
+
+- **Neon Console enhancements**: The table on the [Projects](https://console.neon.tech/app/projects) page in the Neon Console now includes an **Integrations** column that lists your project's integrations. If there are no integrations, an **Add** option takes you to the **Integrations** page where you can view available integrations.
+
+- **Postgres 17 wait events system view support**: Added support for the new [pg_wait_events](https://www.postgresql.org/docs/current/view-pg-wait-events.html) system view introduced in Postgres 17. This view reports wait event types, useful for adding descriptions to wait events reported in `pg_stat_activity`.
```
[the remark this answers is not in the document]
Comment (danieltprice) [reply content/changelog/2024-10-11.md:48]: Thank you @MMeent I'll remove this item now and save your revised version for the changelog that follows the next compute release.

File: content/changelog/2024-10-11.md (hunk 3 above)
Comment (danieltprice) [reply content/changelog/2024-10-11.md:38]: Thank you. I'll remove it.

**Pull request merged**